Jenkinsfile

```diff
@@ -264,8 +264,17 @@ def job = {
                                     --docker-registry ${config.dockerRegistry} \
                                     --project-version ${config.ksql_db_artifact_version} \
                                     --upstream-version ${config.cp_version} --jar
+                                
                                 """
                             }
+                            // Run smoke tests for the packages produced above.  This utilizes
+                            // docker and must be wrapped in a withDockerServer
+                            sh """
+                                echo "Package Smoke tests"
+                                # The last argument is the public CP version to use for dependencies
+                                # such as Zookeeper and Kafka when testing ksqlDB. 
+                                ${env.WORKSPACE}/smoke/run_smoke.sh ${env.WORKSPACE} ./output/ "6.1"
+                            """
```

Comment:
NIT: Not a blocker for my approval, but logically you could instead do this in a different stage called `Test KSQL-DB` (currently in `Build KSQL-DB`), but that would add a lot more lines of code to re-use the deeply nested `with*` directives above. Again not a blocker for me, more of a visual thing when looking at the stages of the build.

Reply:
Ok, made this change.  It actually wasn't bad at all since I didn't need most of those with blocks, just a couple.